### ISSUE TITLE
Linker error on windows

### ISSUE THREAD

**Issue by olilarkin** [Torsion-Audio/nn-inference-template#1]
Following the instructions, I get a linker error, presumably from libTorch trying to find MKL. I have MKL installed. I tried to fix it but no joy yet

LINK : fatal error LNK1181: cannot open input file 'mkl_intel_ilp64.lib'

**Comment by vackva** [comment]
Hi @olilarkin 

you are right, libtorch needs the mkl library. For that the mkl is linked in `cmake/SetupLibTorch.cmake`:

Can you check if the path to your mkl library is different from: `C:/Program Files (x86)/Intel/oneAPI/mkl/2023.0.0/lib/intel64`


`cmake/SetupLibTorch.cmake` relevant code:
```
if (MSVC)
    if (EXISTS "C:/Program Files (x86)/Intel/oneAPI/mkl/2023.0.0/lib/intel64")
        link_directories("C:/Program Files (x86)/Intel/oneAPI/mkl/2023.0.0/lib/intel64")
        get_directory_property(_my_link_dirs LINK_DIRECTORIES)
        message(STATUS "Intel MKL library found at ${_my_link_dirs}")
    endif()
endif (MSVC)
```

**Comment by olilarkin** [comment]
that was it... I have v2023.1.0

**Comment by vackva** [comment]
Ok, thanks for the report! I will look for a more elegant way to import the lib

**Comment by vackva** [comment]
Problem solved in #e7f1828
